<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateStadiumWindow</class>
 <widget class="QWidget" name="CreateStadiumWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>421</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1">
     <widget class="QLineEdit" name="teamNameEditBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="stadiumNameEditBox"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Grass Type</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Stadium Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="addressEditBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="phoneNumberEditBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Team Name</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="leagueTypeComboBox">
      <item>
       <property name="text">
        <string>National League</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>American League</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Seating Capacity</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>League Type</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Openning Date</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="grassTypeComboBox">
      <item>
       <property name="text">
        <string>Grass</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Astro Turf</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="openningDateEditBox"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="seatingCapacityEditBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
